src/viking.js

```diff
@@ -1,11 +1,106 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor (health, strength) {
+        this.health = health
+        this.strength = strength
+    }
+
+attack() {
+    return this.strength
+}
+receiveDamage(damage) {
+    this.health = this.health - damage
+}
```

Comment:
```suggestion
    attack() {
        return this.strength
    }
    receiveDamage(damage) {
        this.health = this.health - damage
    }
```

Soy un pijo pero tabulamenlo!!! si luego le enseñas el codigo asi a Carlos y Julio daran volteretas! :D

Reply:
Gracias!! jajajja